doctor/lib/mojibake.py

```diff
@@ -47,7 +47,6 @@ def fix_mojibake(text):
         "Ò": "N",
         "Ñ": "O",
         "Ð": "P",
-        "": "Q",  # Missing
```

Comment:
Duplicate with line 56

```
doctor/lib/mojibake.py:57:9: F601 Dictionary key literal `""` repeated
   |
55 |         "Ê": "V",
56 |         "É": "W",
57 |         "": "X",  # Missing
   |         ^^ F601
58 |         "Ç": "Y",
59 |         "Æ": "Z",
   |
   = help: Remove repeated key literal `""`
```

https://docs.astral.sh/ruff/rules/multi-value-repeated-key-literal/

Since the last declaration is used, I removed this shadowed one